cmd/uplink/cmd/common_test.go

```diff
@@ -89,11 +89,12 @@ func TestUseOrLoadEncryptionKeyIntoEncryptionAccess(t *testing.T) {
 }
 
 func TestSaveLoadEncryptionKey(t *testing.T) {
-	var inputKey []byte
+	var inputKey string
```

Comment:
This definitely helps with readability in the code! Do you mind to elaborate more on the benefit of initializing the `inputKey` as a string?

Reply:
It isn't about readability of the test, it's because `uplink.SaveEncryptionKey` takes a string as a first parameter https://github.com/storj/storj/pull/2090/files#diff-2b3f299c98b7df4789fbfde5c3e41b40R14